matlab/Ice/context/client.m

```diff
@@ -0,0 +1,59 @@
+% Copyright (c) ZeroC, Inc.
+
+function client(args)
+    % We put the code generated by the Slice compiler in the 'generated' subdirectory.
+    addpath('generated');
```

Comment:
Maybe rmpath would be better handled with a `onCleanup`, can we avoid the `addpath` by generating the code in the current directory, I think we still put everything in VisitorCenter, so is easy to .gitignore it.